tests/x11/remote_desktop/xrdp_server.pm

```diff
@@ -74,13 +74,9 @@ sub run {
     # Wait until xrdp client finishes remote access
     wait_for_children;
 
-    # Gnome on ppc64le needs a bigger timeout
-    if (get_var('OFW')) {
-        send_key_until_needlematch 'displaymanager', 'esc', 20, 5;
-    }
-    else {
-        send_key_until_needlematch 'displaymanager', 'esc';
-    }
+    # We have to click on the mouse before on ppc64le (bug?)
+    mouse_click if get_var('OFW');
+    send_key_until_needlematch 'displaymanager', 'esc';
```

Comment:
Is there any issue if `mouse_click` is also called in other ARCHs?

Reply:
Shouldn't be. It's just as it works well on x86_64 I don't want to change the current behaviour (and apart for SAP, all tests are running only on x86_64 at this time).